### PR TITLE
[docs] Add example requests to API Reference

### PR DESCRIPTION
**Pull request by brzep**
PR for #752 

Added example requests to RTP/MP4/DeckLink pages to make going to HTTP Routes page not necessary.
Requests are split into two code blocks to make copying request json easier.

image

image

### CODE REVIEW

File: docs/pages/api/inputs/rtp.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+      "type": "rtp_stream",
+      "transport_protocol": "tcp_server",
+      "port": 9001,
+      "video": {
+        "decoder": "ffmpeg_h264"
+      },
+      "audio": {
+        "decoder": "opus"
+      },
+      "required": true,
```
Comment (WojciechBarczynski) [inline docs/pages/api/inputs/rtp.md:34]: I think that we should emphasize in docs, that setting `required` to `true` for failable streams will stop the entire processing of the LiveCompositor once such input fails/stops sending data and avoid setting it to `true` in example requests.

**Review by WojciechBarczynski — APPROVED** [review]

**Comment by WojciechBarczynski** [comment]
@brzep BTW you don't have to assign the entire team for review. Usually assigning 1 or 2 team members should be enough.

**Comment by brzep** [comment]
I know I don't have to assign everyone (normally I don't do that), but I assumed it's a very simple change and reviewing it will take like 2 minutes, so more people can do it. I think with the docs the more people see them, possibly the more feedback I'll receive so they'll be more user friendly.

File: docs/pages/api/inputs/decklink.md [hunk 2]
```diff
@@ -11,4 +12,30 @@ import Docs from "@site/pages/api/generated/renderer-DeckLink.md"
 
 An input type that allows consuming streams from Blackmagic DeckLink cards.
 
+### Usage
+
+To use DeckLink Input you must register it first. You can do it by sending a request like this:
```
Comment (wkozyra95) [inline docs/pages/api/inputs/decklink.md:17]:
```suggestion
To use DeckLink input you must register it first. You can do it by sending a request like this:
```

File: docs/pages/api/inputs/decklink.md [hunk 3]
```diff
@@ -11,4 +12,30 @@ import Docs from "@site/pages/api/generated/renderer-DeckLink.md"
...
+    ```
+
+    ```js
+    {
+      "type": "decklink",
+      "subdevice_index": 0,
+      "display_name": "DeckLink Quad HDMI Recorder (3)",
+      "persistent_id": "ffffffff",
+      "enable_audio": false,
+      "required": true,
```
Comment (wkozyra95) [inline docs/pages/api/inputs/decklink.md:34]:
```suggestion
```

File: docs/pages/api/inputs/decklink.md [hunk 4]
```diff
@@ -11,4 +12,30 @@ import Docs from "@site/pages/api/generated/renderer-DeckLink.md"
...
+
+    ```http
+    POST: /api/input/:input_id/register
+    Content-Type: application/json
+    ```
+
+    ```js
+    {
+      "type": "decklink",
+      "subdevice_index": 0,
```
Comment (wkozyra95) [inline docs/pages/api/inputs/decklink.md:30]:
```suggestion
```

File: docs/pages/api/inputs/mp4.md [hunk 5]
```diff
@@ -8,4 +13,29 @@ This input type supports mp4 video tracks encoded with h264 and audio tracks enc
...
+    Content-Type: application/json
+    ```
+
+    ```js
+    {
+      "type": "mp4"
+      "url": "https://url.to.file.mp4",
+      "loop": true,
+      "required": true,
+      "offset_ms": 128
```
Comment (wkozyra95) [inline docs/pages/api/inputs/mp4.md:34]:
```suggestion
```

File: docs/pages/api/inputs/rtp.md [hunk 6]
```diff
@@ -1,6 +1,43 @@
...
+      "offset_ms": 64
```
Comment (wkozyra95) [inline docs/pages/api/inputs/rtp.md:35]:
```suggestion
```

File: docs/pages/api/outputs/mp4.md [hunk 7]
```diff
@@ -11,4 +13,53 @@ import Docs from "@site/pages/api/generated/output-Mp4Output.md"
...
+    ```js
+    {
+      "type": "mp4",
+      "path": "/path/to/file.mp4",
+      "video": {
+        "resolution": { "width": 1280, "height": 720 },
+        "encoder": {
+          "type": "ffmpeg_h264",
+          "preset": "ultrafast",
+          "ffmpeg_options": { "crf": "32" }
```
Comment (wkozyra95) [inline docs/pages/api/outputs/mp4.md:37]:
```suggestion
```

File: docs/pages/api/outputs/mp4.md [hunk 8]
```diff
@@ -11,4 +13,53 @@ import Docs from "@site/pages/api/generated/output-Mp4Output.md"
...
+            "type": "view",
+            "background_color_rgba": "#4d4d4dff"
+          }
+        }
+      },
+      "audio": {
+        "mixing_strategy": "sum_scale",
+        "send_eos_when": {
+          "any_of": [ "input1" ]
+        },
```
Comment (wkozyra95) [inline docs/pages/api/outputs/mp4.md:50]:
```suggestion
```

**Review by wkozyra95 — COMMENTED** [review]
Example requests should include stuff that is required, or optional fields that most use will need to specify. Stuff like offset/required/ffmpeg_options do not help here

File: docs/pages/api/inputs/rtp.md (hunk 1 above)
Comment (wkozyra95) [reply docs/pages/api/inputs/rtp.md:34]: agree, but I don't think setting it to true is the issue here, specifying this field at all in the example is more of a problem

**Pull request merged**